src/widget/systemtrayicon.h

```diff
@@ -22,6 +22,7 @@
 #define SYSTEMTRAYICON_H
 
 #include "systemtrayicon_private.h"
```

Comment:
@agilob Though unrelated to your current changes in this PR. If I understand PIMPL right and not am completely blind, this line should be moved to cpp, to keep implementation **private**. :smile: